R/shiny.R

```diff
@@ -0,0 +1,35 @@
+#' Open an interactive chat application
+#'
+#' Use `chatApp()` to launch an interactive shiny app for any chat instance.
+#'
+#' @param chat A chat object created by [new_chat_openai()] or friends.
+#' @export
+#' @returns A [shiny::shinyApp].
+chatApp <- function(chat) {
```

Comment:
Not `chat_app`? 🤔

Reply:
I was trying to be consistent with shiny 😆

Reply:
In this context, Shiny is just an implementation detail 😄